tests/unit/backend/layers/common/base_api_test.py

```diff
@@ -90,66 +41,9 @@ def _mock_assert_authorized_token(self, token: str, audience: str = None):
             raise Exception()
 
 
-class NewBaseTest(BaseAuthAPITest):
-
-    business_logic: BusinessLogic
-    crossref_provider: CrossrefProviderInterface  # Can be mocked from the tests
-    uri_provider: UriProviderInterface
-
-    sample_dataset_metadata: DatasetMetadata
-
-    @classmethod
-    def setUpClass(cls) -> None:
-        cls.run_as_integration = True if os.environ.get("INTEGRATION_TEST", "false").lower() == "true" else False
-        if cls.run_as_integration:
-            cls.database_provider = DatabaseProvider(database_uri="postgresql://postgres:secret@localhost")
-            cls.database_provider._drop_schema("persistence_schema")
-
+class NewBaseAPITest(BaseAuthAPITest, NewBaseTest):
```

Comment:
Can we rename this to BasePortalAPITest? To remove the 'new' from the name pre-emptively, and also to clarify that this is specific to the PortalAPI tests (assuming curation API will use its own subclass, as the setUp seems portalAPI specific). Otherwise, BaseAPITest is fine if it'll be adapted for both directly.

Reply:
Yes, we absolutely need to do this.